tests/test_utils.py

```diff
@@ -463,8 +463,8 @@ def inverted_dummy_template_operations(wires):
     ops = []
 
     for wire in reversed(wires):
-        ops.append(qml.RY(-1, wires=[wire]).inv())
-        ops.append(qml.RX(1, wires=[wire]).inv())
```

Comment:
Just to understand, .inv() still works but will be deprecated?

Reply:
`qml.inv()`, `op.inv()`, and `qml.adjoint()` should all work identically.

- `qml.inv()` is marked as deprecated (in my head, need to formalize this).
- `op.inv()` is not marked for deprecation, we still need to work out what the future of this is.